PracticeExample/Test.java

```diff
@@ -0,0 +1,10 @@
+package PracticeExample;
+
+public class Test {
+	public static void main(String[] args) {
+		Cylinder c1 = new Cylinder(4, 2);
```

Comment:
use meaningful variable name and not c1, c

Reply:
Adding meaningful name.